tests/test_cli.py

```diff
@@ -32,16 +32,40 @@ def test_bad_args_3(self, runner):
 
     def test_intersects_predicate(self, runner):
         result = runner.invoke(
-            app, ["list-feeds", "-pd", "intersects", "--bbox", "6.626953,49.423342,23.348144,54.265953"]
+            app,
+            ["list-feeds", "-pd", "intersects", "--bbox", "6.626953,49.423342,23.348144,54.265953"],
+            input="N\n",
         )
         assert result.exit_code == 0
 
     def test_contains_predicate(self, runner):
         result = runner.invoke(
-            app, ["list-feeds", "-pd", "contains", "--bbox", "6.626953,49.423342,23.348144,54.265953"]
+            app,
+            ["list-feeds", "-pd", "contains", "--bbox", "6.626953,49.423342,23.348144,54.265953"],
+            input="N\n",
         )
         assert result.exit_code == 0
 
     def test_pretty(self, runner):
-        result = runner.invoke(app, ["list-feeds", "-pt"])
+        result = runner.invoke(app, ["list-feeds", "-pt"], input="N\n")
+        assert result.exit_code == 0
+
+
+class TestFetchFeedsCommand:
+    def test_help(self, runner):
+        result = runner.invoke(app, ["fetch-feeds", "--help"])
+        assert result.exit_code == 0
+        assert "Fetch feeds from sources." in result.stdout
+
+    def test_bad_args(self, runner):
+        result = runner.invoke(app, ["fetch-feeds", "-src", "berlin", "-s", "cdta"])
+        assert result.exit_code == 2
+        assert "Please pass either sources or search" in result.stdout
+
+    def test_fetch_with_sources(self, runner):
+        result = runner.invoke(app, ["fetch-feeds", "-src", "berlin"])
+        assert result.exit_code == 0
```

Comment:
is this downloading? if so, we should probably avoid that and only unit test everything else